lib/plugin/selenoid.js

```diff
@@ -0,0 +1,155 @@
+const util = require('util');
+const axios = require('axios').default;
+const exec = util.promisify(require('child_process').exec);
+const {
+  container, event, recorder, output,
+} = require('../index');
+
+let dockerCreateScript = `docker create --rm --name $name$ -p 4444:4444 -v /var/run/docker.sock:/var/run/docker.sock -v ${global.codecept_dir}/:/etc/selenoid/:ro -v ${global.output_dir}/video/:/opt/selenoid/video/ -v ${global.output_dir}/logs/:/opt/selenoid/logs/ -e OVERRIDE_VIDEO_OUTPUT_DIR=${global.output_dir}/video/ $additionalParams$ aerokube/selenoid:latest-release -log-output-dir /opt/selenoid/logs`;
+let dockerStartScript = 'docker start $name$';
+let dockerStopScript = 'docker stop $name$';
+const seleniumUrl = 'http://localhost:4444';
+
+const createSelenoid = () => exec(dockerCreateScript);
```

Comment:
should we check if the docker is installed and running?

Reply:
@PeterNgTr Not needed actually. When autoStart is enabled we will create docker container with `--rm` which will deleted automatically while exiting the process. So this check is not needed. :)